src/components/functionselect/functionselect.js

```diff
@@ -44,12 +44,15 @@ angular.module('vlui')
             'secondsmilliseconds'
           ]
         };
-        timeUnits.all = timeUnits.aboveFold.concat(timeUnits.belowFold);
+        temporalFunctions.all = _.concat(temporalFunctions.aboveFold, temporalFunctions.belowFold);
 
-        // aggregates for Q
-        var aggregates = {
+        // timeUnits = T functions - undefined
+        var timeUnits = _.pull(temporalFunctions.all, undefined);
```

Comment:
From `pull`'s documentation

> Unlike `_.without`, this method mutates array

I don't understand why you want to mutate temporalFunctions.all.